node/process.ts

```diff
@@ -467,6 +470,10 @@ class Process extends EventEmitter {
     const [prevSec, prevNano] = time;
     return [sec - prevSec, nano - prevNano];
   }
+  
+  memoryUsage() {
+    return Deno.memoryUsage();
+  }
```

Comment:
@AaronO you probably want to add `arrayBuffers` with the value 0

Reply:
Yep, will do. I had suggested adding that field in the original Deno PR